test/wv/html_fixtures/button_go_away.html

```diff
@@ -1,4 +1,6 @@
-<div id="2" style="display:flex;flex-direction:row;flex-wrap:wrap;align-content:flex-start;justify-content:flex-start;align-items:flex-start;width:100%;height:100%"><button id="3" onclick="scarpeHandler('3-click')" onmouseover="scarpeHandler('3-hover')">Go away, button</button>
-  <div id="root-fonts"></div>
-  <div id="root-alerts"> </div>
+<div id="2" style="display:flex;flex-direction:row;flex-wrap:wrap;align-content:flex-start;justify-content:flex-start;align-items:flex-start;width:100%;height:100%">
+  <div style="height:100%;width:100%;position:relative"><button id="3" onclick="scarpeHandler('3-click')" onmouseover="scarpeHandler('3-hover')">Go away, button</button>
```

Comment:
Yeah, I know it generates like 50 more file changes now, but I do like being able to compare the HTML. thanks

Reply:
I agree on the fixtures. Having a really straightforward side-by-side about what changed is *wonderful*. Even better is going through and making sure nothing changed when it shouldn't -- I had to go back and forth on ones of those PRs several times before I got TextDrawables working right again, and the HTML fixtures caught the problem nicely.